components/atom/tooltip/src/index.scss

```diff
@@ -0,0 +1,166 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$fz-atom-tooltip: $fz-m !default;
+$lh-atom-tooltip: $lh-m !default;
+$wlc-atom-tooltip: 4 !default; // -webkit-line-clamp → lines to show
+
+$maw-atom-tooltip: 250px !default;
+$mah-atom-tooltip: 100px !default;
+$c-atom-tooltip: $c-white !default;
+$bgc-atom-tooltip: $c-black !default;
+$bdrs-atom-tooltip: $bdrs-l !default;
+$op-atom-tooltip: 1 !default;
+$pt-atom-tooltip: $p-m !default;
+$pb-atom-tooltip: $p-m !default;
+$pl-atom-tooltip: $p-m !default;
+$pr-atom-tooltip: $p-m !default;
+$m-atom-tooltip: 0 !default;
+$z-atom-tooltip: 1070 !default;
+
+$w-atom-tooltip-arrow: .8rem !default;
+$h-atom-tooltip-arrow: .4rem !default;
+$c-atom-tooltip-arrow: $bgc-atom-tooltip !default;
+
+// Base class
+.sui-AtomTooltip {
+  display: block;
+  font-size: $fz-atom-tooltip;
+  margin: $m-atom-tooltip;
+  position: absolute;
+  // Allow breaking very long words so they don't overflow the tooltip's bounds
+  word-wrap: break-word;
+  z-index: $z-atom-tooltip;
+
+  .arrow {
+    display: block;
+    height: $h-atom-tooltip-arrow;
+    position: absolute;
+    width: $w-atom-tooltip-arrow;
+
+    &::before {
+      border-color: transparent;
+      border-style: solid;
+      content: "";
+      position: absolute;
+    }
+  }
+
+  &-target {  
+    -moz-user-select: none; /* Firefox */
+    -ms-user-select: none; /* Internet Explorer/Edge */
+    -webkit-touch-callout: none; /* iOS Safari */
+    -webkit-user-select: none; /* Safari */
+    user-select: none; /* Non-prefixed version, currently supported by Chrome and Opera */
+  }
+}
+
+.sui-AtomTooltip--top {
+  padding: $h-atom-tooltip-arrow 0;
+
+  .arrow {
+    bottom: 0;
+
+    &::before {
+      border-top-color: $c-atom-tooltip-arrow;
+      border-width: $h-atom-tooltip-arrow ($w-atom-tooltip-arrow / 2) 0;
+      top: 0;
+    }
+  }
+}
+
+.sui-AtomTooltip--right {
+  padding: 0 $h-atom-tooltip-arrow;
+
+  .arrow {
```

Comment:
if you use this, someone would be able to change this styling, we want to avoid it, it has to be encapsulated.

Is arrow a class inside the wrapped component?

Reply:
Yes! this `.arrow` class definition will only be applied under when they're found under a `.sui-AtomTooltip` element. The base component used  (`Tooltip` of reactstrap) doesn't allow in its current version to set a custom className for this _arrow_ element.

For now it can be kept in this way (very few chances for this to cause a problem). 

Meanwhile, an [issue](https://github.com/reactstrap/reactstrap/issues/1117) with its related [PR](https://github.com/reactstrap/reactstrap/pull/1119) has been submitted to the source project.

If this change is accepted we can proceed to set a custom className for this element.